e2e_test/batch/describe_fragments.slt

```diff
@@ -97,8 +97,7 @@ db error: ERROR: Failed to run the query
 
 Caused by:
   Not supported: non shared source has no fragments to describe
-HINT: Use `DESCRIBE` instead.
-
+HINT: USE `DESCRIBE` instead
```

Comment:
Should we tell users to "USE `DESCRIBE` solely without `FRAGMENTS`  instead"?

Reply:
I think we can do:
```
HINT: USE `DESCRIBE` instead of `DESCRIBE FRAGMENTS`
```